x-pack/plugins/observability/server/lib/rules/custom_threshold/lib/create_condition_script.ts

```diff
@@ -19,7 +19,8 @@ export const createConditionScript = (threshold: number[], comparator: Comparato
   }
   if (comparator === Comparator.OUTSIDE_RANGE && threshold.length === 2) {
     return {
-      source: `params.value < params.threshold0 && params.value > params.threshold1 ? 1 : 0`,
```

Comment:
We could also fix it by using `||` instead of `&&`, and it will give exactly the same outcomes.

Reply:
How would it work then, with threshold0 being the start of the range?

Reply:
It will work as expected and as the current solution. Whoever comes to `true` first, `threshold0` or `threshold1`, the rule will trigger an alert.